priv/rel/files/boot

```diff
@@ -273,9 +273,9 @@ case "$1" in
             echo "Unable to write to $PIPE_DIR. Quitting."
             exit 1
         fi
-
+        EXTRA_ARGS_LIST=$3
```

Comment:
This is only going to pass the third argument, not the full argument list. I think what we need to do is emulate how foreground passes extra arguments with `$@`